### DISCUSSION TITLE
[Toogle-group] Icon and label alignment

### DISCUSSION THREAD

**Discussion by aweell** [dxc-technology/halstack-react#655] [Ideas] [answered]
There's a misalignment between the icon and the label in the toggle group

![Screen Shot 2022-02-09 at 12 57 21 PM](https://user-images.githubusercontent.com/44420072/153196886-739fa3e3-a228-4c63-a150-66d982f76697.png)

Fig. - Misalignment between icon and label in the toggle group button

This can be solved by aligning the items to the center:

![Screen Shot 2022-02-09 at 12 57 28 PM](https://user-images.githubusercontent.com/44420072/153197073-3c404289-9014-4531-864f-67a518fd86b3.png)

Fig. - Example using `align-items: center;` 

- [x] Add `align-items: center` to the icon and label container

**Comment by GomezIvann** [comment, marked as answer]
I mark this discussion as answered since this is already implemented and working!
